### PR TITLE
[Frontend] material ui

### PR DESCRIPTION
**Pull request by dyadyaJora**
#33 
Support material ui components
Demo at http://185.185.71.228/

### CODE REVIEW

File: frontend/src/components/ActivityCard/ActivityCard.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.activity-card {
+	height: 100%;
+	position: relative;
+}
+
+.activity-card .activity-card__loader-wrapper {
+	position: absolute;
+    top: 0;
```
Comment (tcibinan) [inline frontend/src/components/ActivityCard/ActivityCard.css:8]: Please unify indentation in the file.

File: frontend/src/components/ActivityCard/ActivityCard.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  render() {
+    const { logo, title, tags, name, error, isLoading } = this.props;
+    let contentInfo;
+
+    if (isLoading) {
+      contentInfo = 'Loading data';
+    } else if (!error) {
+      contentInfo = <ul>
+        <li>Name: { name || '(not set)'}</li>
+        <li>Languages: { tags && tags.length ?
```
Comment (tcibinan) [inline frontend/src/components/ActivityCard/ActivityCard.js:15]: Lets use *tags* rather then *languages* everywhere.

File: frontend/src/containers/ActivityInfo/ActivityInfo.js [hunk 3]
```diff
@@ -1,36 +1,53 @@
...
+            <ActivityCard
+              title="Github Profile"
+              logo={githubLogo}
+              isLoading={info.github.isLoading}
+              name={info.github.name}
+              tags={info.github.languages}
+              error={info.github.error}
+            />
+          </Grid>
+          <Grid item xs={12} sm={4}>
```
Comment (tcibinan) [inline frontend/src/containers/ActivityInfo/ActivityInfo.js]: I just thought that we should hide activity cards for services which are not implemented yet or which analyses have failed.

**Review by tcibinan — CHANGES_REQUESTED** [review]

File: frontend/src/components/ActivityCard/ActivityCard.js (hunk 2 above)
Comment (dyadyaJora) [reply frontend/src/components/ActivityCard/ActivityCard.js:15]: in next request

File: frontend/src/containers/ActivityInfo/ActivityInfo.js (hunk 3 above)
Comment (dyadyaJora) [reply frontend/src/containers/ActivityInfo/ActivityInfo.js]: Not implemented - yes
But i think if we have error - we have to specify error reason(no profile, network error, etc) and show message in card

Comment (tcibinan) [reply frontend/src/containers/ActivityInfo/ActivityInfo.js]: Our goal is to build something that looks like a CV. Probably, it is not a good idea to show errors right there. Probably, some notification will be of help here.

**Pull request merged**